Dockerfile

```diff
@@ -151,21 +155,21 @@ RUN --mount=type=cache,target=/root/.cache/microdnf:rw \
        gcc-c++ \
        python38-devel \
        python38 \
+       # validates if it still needed in the builder iamge.
+       nodejs \
```

Comment:
Is this new? And here it is on the final runtime image, rather than on the builder. From the comment, not sure if this was meant to be in the build stage..

Reply:
it comes from the develop image, the first stage in this Dockerfile, however, the Konflux build starts from stage 2, the builder image. Thus we had to move a few things from develop image to here.

Reply:
I am still not sure if it is really needed or not in the builder image.
Do you think we can remove it and see how the build goes?

Reply:
@MohammadiIram can you please test it (remove the nodejs package)

Reply:
Most likely it is a dev tool. If it hasn't been here in the final image, I don't think that it is needed.

Reply:
right, removed.